### PR TITLE
Delphi compiler support and Pascal example

### PR DESCRIPTION
**Pull request by JulStrat**
I have added simple SDL & GLAD Pascal example and basic Python flake8 check.

### CODE REVIEW

File: glad/lang/pascal/generator.py [hunk 1]
```diff
@@ -277,8 +270,8 @@ def generate_loader(self, features, extensions):
 
     def write_func_definition(self, fobj, func):
         func_name = self.map_func_name(func)
-        fobj.write('  pointer( {0} ) := load(\'{0}\');\n'.format(func_name))
-
+        fobj.write('  {0} := load(\'{0}\');\n'.format(func_name))
+
```
Comment (Dav1dde) [inline glad/lang/pascal/generator.py:274]: Just some unused spaces

**Comment by Dav1dde** [comment]
@JulStrat sorry for letting this sit so long.

Can you:
* remove the github actions file please, I have a jenkins setup for this already
* remove the 4 unused spaces
* squash all the commits into one commit prefixed with `pascal: `, e.g. `pascal: adds delphi compiler support`

**Comment by JulStrat** [comment]
@Dav1dde Good day.

1. remove the github actions file please, I have a jenkins setup for this already
OK. In my TODO.

2. remove the 4 unused spaces
Yes. My bad. TODO. Rawlog attached.

3. squash all the commits into one commit prefixed with pascal: , e.g. pascal: adds delphi compiler support
OK. In my TODO.

Regards.

**Comment by JulStrat** [comment]
@Dav1dde Good day.

1. remove the github actions file please, I have a jenkins setup for this already
OK. In my TODO.

2. remove the 4 unused spaces
Yes. My bad. TODO. Rawlog attached.

3. squash all the commits into one commit prefixed with pascal: , e.g. pascal: adds delphi compiler support
OK. In my TODO.

Regards.

**Comment by JulStrat** [comment]
@Dav1dde Good day.

1. remove the github actions file please, I have a jenkins setup for this already
OK. In my TODO.

2. remove the 4 unused spaces
Yes. My bad. TODO. Rawlog attached.

3. squash all the commits into one commit prefixed with pascal: , e.g. pascal: adds delphi compiler support
OK. In my TODO.

Regards.

**Comment by JulStrat** [comment]
Done.

File: example/pascal/sdl_glad.pas [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    glBegin(GL_TRIANGLES);
+    glVertex2f(0.2, 0);
+    glVertex2f(0.01, 0.2);
+    glVertex2f(-0.2, 0);
+    glEnd();
+
+    SDL_GL_SwapWindow(window);
+  end;
+
+  (* Destroy everything to not leak memory. *)
```
Comment (dpethes) [inline example/pascal/sdl_glad.pas:86]: Dispose(event); is missing.

Comment (JulStrat) [reply example/pascal/sdl_glad.pas:86]: Yes. I will add Dispose(event) and squash.

Comment (JulStrat) [reply example/pascal/sdl_glad.pas:86]: Fixed. Thanks.

**Comment by Dav1dde** [comment]
@JulStrat thanks for your time and contribution!

**Comment by JulStrat** [comment]
My pleasure.
Best regards.

**Pull request merged**